recipes/cppimport/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cppimport" %}
+{% set version = "22.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bbb4957102db41bc99ad72c233bce92f9d1fd91be352fc07878c4361033a401f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - filelock
+    - mako
+    - pybind11
+    - python >=3.7
+    - setuptools
+
+test:
+  source_files:
+    - tests
+  imports:
+    - cppimport
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v tests/test_cppimport.py  # [not win]
```

Comment:
upstream says it can work on Windows, but CI doesn't run on Windows, so skip pytest instead of declaring it incompatible